src/main/resources/openapi/retentions.yaml

```diff
@@ -164,6 +164,37 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/RetentionPolicyType'
+        '409':
+          description: Conflict
+          content:
+            application/json+problem:
+              schema:
+                $ref: './problem.yaml'
+              examples:
+                non_unique_policy_name:
+                  summary: Non unique policy name
+                  description: Policy name must be unique
+                  value:
+                    type: "RETENTION_109"
+                    title: "The policy name already exists"
+                    status: 409
+                    detail: Policy name "my retention policy" already exists
```

Comment:
We dont set the json detail key. It may be ok to leave it though seeing as its an example